src/main/java/Main.java

```diff
@@ -1,8 +1,51 @@
-public class Main {
 
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Main {
     public static void main(String[] args) {
-        // ваш код начнется здесь
-        // вы не должны ограничиваться только классом Main и можете создавать свои классы по необходимости
-        System.out.println("Привет Мир");
+
+        Scanner scan = new Scanner(System.in);
+        ArrayList<Car> cars = new ArrayList<>();
+        for (int i = 0; i < 3; i++) {
+            while (true) {
+                System.out.println("Введите марку машины");
+                String model = scan.next();
+                System.out.println("Введите скорость машины");
+                Integer speed = scan.nextInt();
+                if (speed >= 240 || speed <= 0) {
```

Comment:
```suggestion
                if (speed > 250 || speed < 0) {
```
Так будет соответствовать условию задачи

Reply:
Исправлено